global/scripts/RSNregression.m

```diff
@@ -0,0 +1,296 @@
+function RSNregression(InputFile, InputVNFile, GroupMaps, Method, ParamsFile, VAWeightsName, OutputBeta, varargin)
```

Comment:
Can you add a usage statement, so that users reading the function can know what the inputs are, and what the `varargin` options are?